decidim-accountability/app/controllers/decidim/accountability/admin/results_controller.rb

```diff
@@ -56,8 +56,32 @@ def destroy
           redirect_to results_path(parent_id: result.parent_id)
         end
 
+        def proposals
+          respond_to do |format|
+            format.html do
+              render partial: "proposals"
+            end
+            format.json do
+              query = Decidim.find_resource_manifest(:proposals)
+                             .try(:resource_scope, current_feature)&.order(title: :asc)
+              term = params[:q]
+              if term&.start_with?("#")
+                term.delete!("#")
+                query = query.where("CAST(id AS TEXT) LIKE ?", "#{term}%")
+              else
+                query = query.where("title ilike ?", "%#{params[:q]}%")
+              end
+              render json: query.all.collect { |p| [p.title, p.id] }
+            end
+          end
+        end
+
+        #-----------------------------------------------------------------------
+
         private
 
+        #-----------------------------------------------------------------------
```

Comment:
Remove this

Reply:
why? It facilitates locating code inside artifacts!

Reply:
Now there's no need to have this here 👍

Reply:
Aaand this :)